src/main/java/taxi/service/DriverService.java

```diff
@@ -3,4 +3,5 @@
 import taxi.model.Driver;
 
 public interface DriverService extends GenericService<Driver> {
+    Driver getByLogin(String login);
```

Comment:
I think it solve few troubles in the `public Driver login(String login, String password)`
```suggestion
    Optional<Driver> getByLogin(String login);
```